pkg/chaosctl/debug/iochaos/iochaos.go

```diff
@@ -57,21 +58,23 @@ func Debug(ctx context.Context, chaos runtime.Object, c *cm.ClientSet, result *c
 func debugEachPod(ctx context.Context, pod v1.Pod, daemon v1.Pod, chaos *v1alpha1.IoChaos, c *cm.ClientSet, result *cm.PodResult) error {
 	// print out debug info
 	cmd := fmt.Sprintf("cat /proc/mounts")
-	out, err := cm.ExecBypass(ctx, pod, daemon, cmd, c.KubeCli)
+	out, err := cm.ExecBypass(ctx, pod, daemon, cmd, c.KubeCli, bpm.PidNS)
 	if err != nil {
 		return fmt.Errorf("run command '%s' failed with: %s", cmd, err.Error())
 	}
 	result.Items = append(result.Items, cm.ItemResult{Name: "mount information", Value: string(out)})
 
 	pids, commands, err := cm.GetPidFromPS(ctx, pod, daemon, c.KubeCli)
+	if err != nil {
+		return fmt.Errorf("get pid from ps failed with: %s", err.Error())
+	}
 
 	for i := range pids {
 		cmd = fmt.Sprintf("ls -l /proc/%s/fd", pids[i])
 		out, err = cm.ExecBypass(ctx, pod, daemon, cmd, c.KubeCli)
-		if err != nil {
-			return fmt.Errorf("run command '%s' failed with: %s", cmd, err.Error())
+		if err == nil {
+			result.Items = append(result.Items, cm.ItemResult{Name: fmt.Sprintf("file discriptors of PID: %s, COMMAND: %s", pids[i], commands[i]), Value: string(out)})
```

Comment:
If the error occurs, the user/developer could not receive any messages. It makes confusing while using and debugging.

Please print this error.

Reply:
Since some process would not have its file descriptors, so I add this error handling method.

Changed to dealing with this certain error separately. Thanks!

Reply:
I think even "No such file or directory" also should be printed. 🤔 

I think "No such file or directory" will occur when some process was killed as soon as `ls /proc/%d/fd`, I think it's also a useful message.

Reply:
Cool that indeed makes sense. Fixed